### ISSUE TITLE
Prop `items` in `Menu` conflicts with props: `children`

### ISSUE THREAD

**Issue by JT501** [Semantic-Org/Semantic-UI-React#4098]
### Steps
In Sidebar component, using as Menu and pass items shorthand to `items` props.

```tsx
const items = [{ name: 'Home' }, { name: 'About' }]

<Sidebar
  as={Menu}
  animation="push"
  vertical
  inverted
  onHide={() => setSideBarOpen(false)}
  visible={sideBarOpen}
  items={items}
/>
```

### Expected Result
It should show items in menu.

### Actual Result
No item is shown in menu. And have following warning:
```
Warning: Failed prop type: Prop `items` in `Menu` conflicts with props: `children`. They cannot be defined together, choose one or the other.
```

### Version
2.0.1

### Testcase
https://codesandbox.io/s/runtime-wood-pgpkp?file=/src/App.tsx

**Comment by layershifter** [comment]
@JT501 thanks for reporting, I fixed it in #4145.

**Comment by layershifter** [comment]
Was fixed in `semantic-ui-react@2.0.3` 🎉
